packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -26,7 +26,6 @@ import {
 } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
 import { join } from 'path';
 import * as vscode from 'vscode';
-import { BaseDeployExecutor } from '.';
```

Comment:
So doing an import like this results in scanning all things exported from the index.ts file.  Which can lead to complications in unit testing.  Updated to pull directly from the file where exported instead.